### ISSUE TITLE
RPi Zero W Support

### ISSUE THREAD

**Issue by kubo6472** [kerberos-io/kerberos-balena#3]
please add support for RPi Zero W device, thanks.

**Comment by kubo6472** [comment]
Adding Logs: https://hastebin.com/itedavuyag.cs
According to your site, you do support RPi Zero W, so why don't you have the RPi Zero W compatible image on Docker Hub?

**Comment by cedricve** [comment]
thank you for reporting @kubo6472 we will create an image. Any help is of course possible. 

https://github.com/kerberos-io/docker

**Comment by kubo6472** [comment]
I was trying to do something. Forked the kerberos-docker image and tried building it from balena provided Debian stretch, compatible with armv6, but got stuck on the PHP packages. But at least I learned something on the way. I'll wait for your image then.

**Comment by cedricve** [comment]
yes just checked your fork, actually we are using `docker buildx` to do our cross builds, will check if they have the armv6 build available. Keep you posted.

**Comment by kubo6472** [comment]
On the [docker blog](https://www.docker.com/blog/multi-arch-images/) they list ARMv6 in the examples. I'll try the buildx then.
![image](https://user-images.githubusercontent.com/26146502/91714528-45f73880-eb8c-11ea-8f5b-50910d51fe1d.png)

**Comment by cedricve** [comment]
We are working on it already @kubo6472 
https://github.com/kerberos-io/kerberos-docker/issues/23

**Comment by iFredLouzada** [comment]
im a noob but looks like i am looking for the same thing 👍

**Comment by kubo6472** [comment]
If you have raspberry pi zero (w or not) then php packages are unsupported. So that's where we're all stuck.

**Comment by iFredLouzada** [comment]
Thank you sir. 
> If you have raspberry pi zero (w or not) then php packages are unsupported. So that's where we're all stuck.

**Comment by carriba** [comment]
Has anybody tried to manually install [KerberosIO](https://doc.kerberos.io/opensource/installation-advanced#raspbian) under Raspbian? 

I have managed to install PHP7 under the latest release of [Raspbian Stretch](http://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/) for another project when using a Pi Zero. 

Not sure to whch extent there may be other dependencies...

**Comment by kubo6472** [comment]
I wonder now, how is this handled in the KiOS? I installed that on my Zero W successfully, so I know it works somehow.
